libraries/Microsoft.Bot.Builder.Core.Extensions/ITranscriptStore.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Microsoft.Bot.Schema;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Bot.Builder.Core.Extensions
+{
+
+    public class Conversation
```

Comment:
>Conversation [](start = 17, length = 12)

Seems like this should be something more specific like a `ConversationTranscript` or something.  Given that it's own namespace are we sure this will be the only `Conversation` object we will ever need?